docs/interpreting-experiments/experiment-progress-bar.md

```diff
@@ -18,7 +20,29 @@ You can also see the progress bar on the details page for each experiment.
 
 **Note:** We compute the days remaining using a linear interpolation. This interpolation does not take into account that gathering data usually slows down during an experiment, and so the estimate may be optimistic, especially in the early days of an experiment.
 
-#### How we calculate the progress
+### How to use the progress bar
+
+Traditionally, when using a fixed sample test, you decide up front how long the experiment ought to run and cannot interpret results until you have finished gathering all data. However, the sequential testing approach we use allows for flexibility in deciding when to stop an experiment. Here's some advice on how to get the most out of the progress bar.
+
+Before starting with any experiment, we have to set the minimum detectable effect to something that makes sense for your use case. Two aspects are important:
```

Comment:
Flipping between "we" and "you" here when I think both should have the same antecedent? So "Before starting the experiment, you should set the minimum detectable effect to something that makes sense for your use case." Also, if we spelled out and then abbreviated "MDE" above, why not use the abbreviation here?

Reply:
Good call out; I'm going to stick with we/our